### PR TITLE
target dummies on nukie planet

### PR DESCRIPTION
**Pull request by Cheackraze**
## About the PR 

**Changelog**

- added target dummies to nukie planet
- that's it

### CODE REVIEW

File: Resources/Maps/nukieplanet.yml [hunk 1]
```diff
@@ -14740,6 +14760,8 @@ entities:
 - uid: 1617
   type: VendingMachineClothing
   components:
+  - flags: SessionSpecific
+    type: MetaData
```
Comment (metalgearsloth) [inline Resources/Maps/nukieplanet.yml:14764]: ???

**Comment by metalgearsloth** [comment]
This has some highly sussy metadata

**Comment by ElectroJr** [comment]
Previously metadata flags weren't being saved, now they are (added to make it easier to deal with initializing entities inside of containers).

As to why shit like vending machines has session specific data, I guess its the actions comp bc sentient vending machines have the ability to dispense stuff?

**Comment by Cheackraze** [comment]
all i did was mapping 11 maps/nukieplanet.yml, and then added the 3 target dummies, and then savegrid 11 maps/nukieplanet.yml

any of the other things was the client changing them

**Pull request merged**